src/app/modules/payments/payments.service.ts

```diff
@@ -50,7 +50,9 @@ export const findPaymentById = (
       paymentId,
       null,
       // readPreference from transaction isn't respected, thus we are setting it on operation
-      session ? { session, readPreference: 'primary' } : null,
+      session
```

Comment:
Issue: seems like there is a ternary where both paths lead to the same outcome.

Seems like it might add confusion for future developers looking at this code.

Reply:
Prefer if the ternary expression is removed entirely